### ISSUE TITLE
 DL4006 with BusyBox ash

### ISSUE THREAD

**Issue by vitaliytv** [hadolint/hadolint#250]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

When using SHELL ["/bin/sh", "-o", "pipefail", "-c"], the commands in a RUN should not be fail with DL4006

### Actual behavior

/dev/stdin:26 DL4006 Set the SHELL option -o pipefail before RUN with a pipe in it
Exited with code 1

### Steps to reproduce the behavior

Last docker version

```bash
Haskell Dockerfile Linter v1.10.4-3-g20c5c3b
```

Dockerfile for reproduce:

```Dockerfile
FROM alpine:3.8

SHELL ["/bin/sh", "-o", "pipefail", "-c"]

RUN curl https://get.acme.sh | sh
```

**Comment by zemanlx** [comment]
That's tricky. `hadolint` have no idea that `/bin/sh` is pointing to `/bin/busybox` which supports `-o pipefail`. 

I am not able to image how we can solve this. I would recommend to ignore it in comment. Any other ideas @lorenzo?

**Comment by matrixik** [comment]
Why not using:
```
SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
```

**Comment by vitaliytv** [comment]
@matrixik  offer solution, thanks
